client/components/Pipeline/Column.tsx

```diff
@@ -53,12 +54,21 @@ export default class Column extends React.Component<ColumnProps, ColumnState> {
     };
   }
 
-  renderCard = (company, pipe, index, status) => {
-    const shouldDisplay = company.pointPartnersNames.has(
+  renderCard = (company, pipe, index, status, userPartnerTeam) => {
+    const selectedPartnerHasCompany = company.pointPartnersNames.has(
       pipe.state.currentPartner
     );
-    const shouldDisplay2 = shouldDisplay || pipe.state.currentPartner === 'ALL';
-    if (shouldDisplay2) {
+    // TODO: Should not filter this way. Instead, filter the companies shown in
+    // the db query on line 76 in Kanban.tsx
```

Comment:
Can you add a general comment of what the filtering process is right now? Can you also add a ticket for this TODO?

Reply:
Done